app/src/main/java/com/gdsctsec/smartt/ui/main/HomeScreenFragment.kt

```diff
@@ -61,15 +70,23 @@ class HomeScreenFragment : Fragment() {
             Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
         )
         titleTextView.append(wordThree)
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-        val timeList: List<String> = listOf(
-            "10:00 - 12:00",
-            "12:00 - 14:00",
-            "14:00 - 16:00",
-            "16:00 - 18:00",
-            "08:00 - 10:00"
-        )
-        val subjectList: List<String> = listOf("Biology", "Math", "Java", "Science", "Python")
+
+        val timeList: MutableList<String> = mutableListOf("0:00 - 0:00")
+        val subjectList: MutableList<String> = mutableListOf("No Lectures as of now")
+
+        val viewModel = HomeScreenViewModel(requireActivity())
+
+        viewModel.getLiveLectureData().observe(requireActivity(), Observer {
+            if(it.size!=0){
+                for(i in 0..it.size-1){
+                    subjectList.add(i, it.get(i).lec)
+                    timeList.add(i, (it.get(i).startTime+" - "+it.get(i).endTime))
+                }
+            }
+        })
+        //mutableListOf("subjects")
         recyclerView = view.findViewById(R.id.home_recyclerView)
         recyclerView.adapter = SubjectsAdapter(subjectList, timeList)
         recyclerView.layoutManager = LinearLayoutManager(requireActivity())
```

Comment:
why is the `super.onViewCreated(view, savedInstanceState)` in the bottom!?, it should be the first line of the method!